src/model/element-base.ts

```diff
@@ -44,7 +44,15 @@ export abstract class ScannedElementBase implements Resolvable {
   superClass?: ScannedReference = undefined;
 
   applyHtmlComment(commentText: string|undefined) {
-    this.description = this.description || commentText || '';
+    if (commentText) {
+      const commentJsdoc = jsdoc.parseJsdoc(commentText);
+      this.jsdoc =
+          this.jsdoc ? jsdoc.join(this.jsdoc, commentJsdoc) : commentJsdoc;
+      this.description = [
+        commentJsdoc.description || '',
+        this.description || ''
+      ].join('\n\n').trim();
+    }
```

Comment:
My read of Justin's position was in slack was that he only holds a weak objection against merging, but a strong preference for warning when there's both. That seems reasonable to me, and that merging also gets us unblocked for the site.

I think the logic would be go here and would be pretty easy, just do: `this.warnings.push({...}` if this.jsdoc has a description or any tags.